### PR TITLE
GUI-2286 Complete integration of tag editor into scaling group view

### PR DESCRIPTION
**Pull request by coyote240**
* Minor changes to tag-editor directive to better integrate into form-level validation.
* Parameterize tag-editor for autoscale.
* Create chosen directive to allow the chosen library to better integrate with angular and actually work with angular form validation.  This will move to it's own module in the next integration PR.
* Update template to inject form data into angular-land.

### CODE REVIEW

File: eucaconsole/static/js/pages/scalinggroup.js [hunk 1]
```diff
@@ -48,7 +87,9 @@ angular.module('ScalingGroupPage', ['TagEditorModule', 'EucaConsoleUtils'])
             // scalingGroupName, policiesCount
             $scope.scalingGroupName = options.scaling_group_name;
             $scope.policiesCount = options.policies_count;
```
Comment (coyote240) [inline eucaconsole/static/js/pages/scalinggroup.js]: Get the data onto the model earlier so angular can work with it.

File: eucaconsole/templates/scalinggroups/scalinggroup_view.pt [hunk 2]
```diff
@@ -39,10 +39,12 @@
                 </metal:block>
                 <form id="scalinggroup-detail-form" action="${request.route_path('scalinggroup_update', id=scaling_group.name)}"
                       method="post" data-abide="" name="scalingGroupDetailForm" novalidate=""
-                      tal:define="avail_zone_html_attrs {'data-placeholder': avail_zone_placeholder_text};
```
Comment (coyote240) [inline eucaconsole/templates/scalinggroups/scalinggroup_view.pt]: Chosen params are passed in as html attributes.

File: eucaconsole/templates/scalinggroups/scalinggroup_view.pt [hunk 3]
```diff
@@ -70,7 +72,7 @@
                     </div>
                     <div tal:condition="not vpc_network">
                         <input type="hidden" name="vpc_subnet" value="None" />
-                        ${panel('form_field', field=edit_form['availability_zones'], leftcol_width_large=3, rightcol_width_large=9, **avail_zone_html_attrs)}
```
Comment (coyote240) [inline eucaconsole/templates/scalinggroups/scalinggroup_view.pt]: Without an ng-model attribute, angular wasn't seeing these values.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1228/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1233/
Test PASSed.

**Pull request merged**